operate/qa/integration-tests/src/test/java/io/camunda/operate/schema/IndexOldSchemaValidatorIT.java

```diff
@@ -260,6 +261,7 @@ private String getFullQualifiedIndexName(final IndexDescriptor index, final Stri
   }
 
   private String getFullQualifiedIndexName(final String indexNamePart, final String version) {
-    return String.format("%s-%s-%s_", operatePrefix, indexNamePart, version);
+    return String.format(
```

Comment:
This fails when the prefix is empty or null right?

Reply:
yes, this test class is always using a prefix though

Reply:
will use the same formatter here too